.github/workflows/gradle-build-test.yaml

```diff
@@ -24,8 +23,6 @@ jobs:
 
             -   name: Build with Gradle Wrapper
                 env:
-                    APP_SCHEDULING_WORKFLOWRUNSINTERVAL: '30 */5 * * * ?'
-                    APP_SCHEDULING_JOBSINTERVAL: '0 */1 * * * ?'
                     APP_GITHUB_APPLICATION_ID: '882159'
```

Comment:
not your fault but noticed from previous already. these should probably be .env variables